Gemfile

```diff
@@ -44,7 +44,7 @@ gem 'execjs'
 gem 'therubyracer'
 gem 'omniauth-facebook'
 gem 'ancestry'
-
+gem "breadcrumbs_on_rails"
```

Comment:
Do we need this gem? It looks like functionality it provides is never used in views. Am I wrong?